### PR TITLE
Store configuration inside a JSON file

### PR DESCRIPTION
**Pull request by ryzokuken**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
   "license": "GPL-3.0",
   "dependencies": {
     "irc": "^0.5.2",
+    "jsonfile": "^3.0.0",
```
Comment (ananyo2012) [inline package.json]: We can use the inbuilt `fs` module instead of using of adding an extra dependency isn't it?

**Comment by ryzokuken** [comment]
If we do it that way, we would have to use fs, read a string and then use
JSON.parse(). This module does that for us, leaving us to do the relatively
simple stuff.

File: package.json (hunk 1 above)
Comment (ryzokuken) [inline package.json]: If we do it that way, we would have to use fs, read a string and then use
JSON.parse(). This module does that for us, leaving us to do the relatively
simple stuff.

Comment (ryzokuken) [inline package.json]: @ananyo2012 If you like, I could do it by using `fs` and put the code inside a function in `utils`.

Comment (ananyo2012) [inline package.json]: @ryzokuken Its same 2 lines of code. You don't need to put it in utils. See https://stackoverflow.com/a/10011078

**Comment by ryzokuken** [comment]
@ananyo2012 Done, thanks.

**Comment by ananyo2012** [comment]
Ah the last build failed. Error in downloading yarn.

**Pull request merged**